app/models/survey.py

```diff
@@ -0,0 +1,45 @@
+from .db import db
+
+# this is the through table
+answers = db.Table("answers", db.Model.metadata,
+    db.Column("requestId", db.Integer, db.ForeignKey("requests.id"), primary_key=True),
+    db.Column("questionId", db.Integer, db.ForeignKey("questions.id"), primary_key=True),
+    db.Column("answer", db.String, nullable=False))
```

Comment:
I think for the through table we only need the data for the two classes we are trying to tie together. So maybe we can get rid of line 7

Reply:
line 7 is needed so we have a place to hold the answers between the request and question.